package.json

```diff
@@ -18,11 +18,13 @@
     "jss-vendor-prefixer": "^7.0.0",
     "lodash": "^3.10.0",
     "material-ui": "^1.0.0-beta.22",
+    "mdurl": "^1.0.1",
     "normalize-jss": "^3.1.0",
     "prop-types": "^15.5.10",
     "react": "^15.3.0",
     "react-dom": "^15.3.0",
     "react-jss": "^8.3.3",
+    "react-router-dom": "^4.2.2",
```

Comment:
move everything router related here from the client and export from this package